api/ProgressEvent.json

```diff
@@ -94,6 +93,45 @@
           }
         }
       },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "support": {
+            "chrome": {
+              "version_added": "38",
+              "partial_implementation": true,
+              "notes": "Available in dedicated workers and shared workers, not available in service workers. See <a href='https://crbug.com/332663431'>bug 332663431</a>."
+            },
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/API/Navigator/serviceWorker#browser_compatibility, Chrome only supports service workers from version 40. Can you clarify?

Reply:
![image](https://github.com/user-attachments/assets/0c004f9a-841a-4c7e-811c-643f9fdf491e)

confirmed 47 is the correct version